popups.py

```diff
@@ -0,0 +1,44 @@
+import folium
+import pandas as pd
+import streamlit as st
+from streamlit_folium import st_folium
+from folium.plugins import MarkerCluster
+import math
+
+# these are the CSV files of the capitals and countries, both located in data
+states = pd.read_csv(
+    r"data/us-state-capitals.csv")
+counties = pd.read_csv(
+    r"data/uscounties.csv")
+
+# creates the Streamlit map and the add to the clusters map
+map = folium.Map()
+marker_cluster = MarkerCluster().add_to(map)
+
+# helper to create the markers
+def create_markers(df, lat_col, lon_col, info_col, markers):
+    '''
+    Creates markers based on information in df specified by the lat_col and 
+    lon_col to reference in the df. The marker will also have a popup that 
+    displays the information that corresponds to it in the df. 
+
+    REQUIRES
+    - df that is a dataframe that has colums with lon_col, lat_col, and info_col
+    - lat_col is a string that represents the lat_col in the df, which contains 
+    the latitude coordinate
+    - lon_col is a string that represents the lon_col in the df, which contains 
+    the longitude coordinate
+    - info_col is a string that represents the info_col in the df, which contains
+    the information (such as the name)
+    '''
+    for index, row in df.iterrows():
```

Comment:
since you're not using index anywhere, if there's a way to iterate over the rows without it, it would clean up this loop. if not, ignore this :)